src/registrar/assets/sass/_theme/_uswds-theme-custom-styles.scss

```diff
@@ -422,3 +422,9 @@ abbr[title] {
   border-bottom: none;
   text-decoration: none;
 }
+
+.usa-textarea {
+  @include at-media('tablet') {
+    height: 20rem;
```

Comment:
I've been trying to avoid unnamed space and size units throughout the style sheet as best I could. Could we use a USWDS unit here? For 20 rem you could use `units('mobile')`

Reply:
10-4